plan.sh

```diff
@@ -1,16 +1,24 @@
 pkg_name=erlang18
 pkg_origin=core
-pkg_version=18.3
+pkg_version=18.3.4.11
 pkg_description="A programming language for massively scalable soft real-time systems."
 pkg_upstream_url="http://www.erlang.org/"
 pkg_dirname=otp_src_${pkg_version}
 pkg_license=('Apache-2.0')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
-pkg_source=http://www.erlang.org/download/otp_src_${pkg_version}.tar.gz
+pkg_source=https://api.github.com/repos/erlang/otp/tarball/refs/tags/OTP-${pkg_version}
```

Comment:
Why are we changing the package source?

Reply:
webpage for erlang v18.3.4.11 (https://www.erlang.org/patches/otp-18.3.4.11) suggest download link as mentioned in pkg_source. https://www.erlang.org/download does not have v18.3.4.11 available for download.